content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -7,11 +7,10 @@ aliases:
 keywords: [traffic-management,egress]
 ---
 
-By default, Istio-enabled services are unable to access URLs outside of the cluster because the pod uses
-iptables to transparently redirect all outbound traffic to the sidecar proxy,
-which only handles intra-cluster destinations.
+By default, Istio-enabled services are able to access URLs outside of the cluster.  If you choose to tighten up the outbound access control to services running outside of the cluster, you can configure `global.outboundTrafficPolicy.mode` to `REGISTRY_ONLY`.  In this `REGISTRY_ONLY` mode, Istio configures the pod to use iptables to transparently redirect all outbound traffic to the sidecar proxy, which only handles intra-cluster destinations.
+
 This task describes how to configure Istio to expose external services to Istio-enabled clients
-in three different ways:
+in three different ways, when `REGISTRY_ONLY` mode is enabled:
```

Comment:
@linsun So it is two ways. The third ways was to disable `REGISTRY_ONLY`.

Reply:
@linsun I mean there are three ways, one of them is the default, when `REGISTRY_ONLY` mode is disabled.

Reply:
This sentence should not be changed. The task shows 3 different ways period. The fact that the third way is enabled by default is not relevant here.

Reply:
OK, I see your suggestion, "
Depending on your Istio configuration, Istio-enabled services may not be able to access URLs outside of the cluster", together it is clear for me.